tests/test_major_version/expected.nix

```diff
@@ -0,0 +1,30 @@
+# file generated from go.mod using vgo2nix (https://github.com/adisbladis/vgo2nix)
+[
+  {
+    goPackagePath = "github.com/cespare/xxhash/v2";
+    fetch = {
+      type = "git";
+      url = "https://github.com/yihuang/test-golang-module-major-version";
+      rev = "4a13e4bfd5a9";
+      sha256 = "0i1bpw3v73jhwnpa8vv276qhy39fk6qhgyci0wng9mwr2pxfha5y";
+    };
+  }
+  {
+    goPackagePath = "github.com/yihuang/test-golang-module-major-version";
+    fetch = {
+      type = "git";
+      url = "https://github.com/yihuang/test-golang-module-major-version";
+      rev = "sub/v2.0.1";
```

Comment:
I think the expected here should be "v1.0.1"

Reply:
You are right, fixed this. https://github.com/nix-community/vgo2nix/compare/7feab3f223f17f683ef555860aa847a111a2b31c..57014aa18aef8ae303df07da1c46a9a6777dac6b